litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/index.tsx

```diff
@@ -132,18 +133,17 @@ const WorkflowNodeInfo: React.FC<WorkflowNodeInfoProps> = ({
               </Button>
               {isAppInfoVisible && (
                 <Typography className={classes.textMargin}>
-                  {embeddedYAMLString &&
-                    Object.keys(
-                      YAML.parse(embeddedYAMLString).spec.appinfo
-                    ).map((key, index) => (
+                  {Object.keys(YAML.parse(embeddedYAMLString).spec.appinfo).map(
+                    (key, index) => (
                       <div key={index.toString()}>
                         <strong>{key} :</strong>
```

Comment:
use a style={{fontWeight: 'bold'}} in the above div @Jonsy13

Reply:
I don't think there is a problem with this, It helps to develop the code continuously without going to CSS file just for one CSS property. If it were 2 properties or more I would have made 1 CSS class, but here I don't it is necessary.

Reply:
please remove the jsx @Jonsy13 no need of any class provide an inline style above to this `<div key={index.toString()}>` please do not add extra class.

Reply:
If I add an inline style to `<div key={index.toString()}>`, then the key-value pair both will be bold, which is not the design requirement.

Reply:
sure then this is good.